### PR TITLE
Bump `open-aea` and `open-autonomy` frameworks

### PR DESCRIPTION
**Pull request by Adamantios**
## Proposed changes

Bumps `open-aea` and `open-autonomy` frameworks.

## Fixes

If it fixes a bug or resolves a feature request, be sure to link to that issue.

## Types of changes

What types of changes does your code introduce? (A **breaking change** is a fix or feature that would cause existing functionality and APIs to not work as expected.)
_Put an `x` in the box that applies_

- [ ] Non-breaking fix (non-breaking change which fixes an issue)
- [ ] Breaking fix (breaking change which fixes an issue)
- [ ] Non-breaking feature (non-breaking change which adds functionality)
- [ ] Breaking feature (breaking change which adds functionality)
- [ ] Refactor (non-breaking change which changes implementation)
- [ ] Messy (mixture of the above - requires explanation!)

## Checklist

_Put an `x` in the boxes that apply._

- [x] I have read the [CONTRIBUTING](../blob/main/CONTRIBUTING.md) doc
- [x] I am making a pull request against the `main` branch (left side). Also you should start your branch off our `main`.
- [x] Lint and unit tests pass locally with my changes
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have locally run services that could be impacted and they do not present failures derived from my changes

## Further comments

n/a

### CODE REVIEW

**Review by 0xArdi — APPROVED** [review]

File: packages/valory/skills/oracle_abci/composition.py [hunk 1]
```diff
@@ -68,17 +66,9 @@
...
-        "safe_contract_address",
-    ]
-)
-
 AgentRegistrationAbciApp.db_post_conditions[FinishedRegistrationRound].extend(
     [
+        get_name(ODSynchronizedData.oracle_contract_address),
         # this is a patch and needs to be addressed in the autonomy repo
         # we need to add this in the post-conditions of the registration abci
         "safe_contract_address",
```
Comment (DavidMinarsch) [inline packages/valory/skills/oracle_abci/composition.py:74]: This can be removed

File: packages/valory/skills/oracle_abci/skill.yaml [hunk 2]
```diff
@@ -122,6 +122,8 @@ models:
       service_id: price_estimation
       service_registry_address: null
       setup:
+        all_participants:
+        - {}
```
Comment (DavidMinarsch) [inline packages/valory/skills/oracle_abci/skill.yaml:126]: Suggest to set a single zero address

File: packages/valory/skills/price_estimation_abci/skill.yaml [hunk 3]
```diff
@@ -126,6 +126,8 @@ models:
       service_id: price_estimation
       service_registry_address: null
       setup:
+        all_participants:
+        - {}
```
Comment (DavidMinarsch) [inline packages/valory/skills/price_estimation_abci/skill.yaml:130]: see above

File: packages/valory/skills/oracle_abci/skill.yaml (hunk 2 above)
Comment (Adamantios) [reply packages/valory/skills/oracle_abci/skill.yaml]: aee585279e78ca23e328cf4428015809f2ddfe69

File: packages/valory/skills/price_estimation_abci/skill.yaml (hunk 3 above)
Comment (Adamantios) [reply packages/valory/skills/price_estimation_abci/skill.yaml]: aee585279e78ca23e328cf4428015809f2ddfe69

**Pull request merged**